sdk/servicebus/service-bus/test/perf/rhea-promise/send.ts

```diff
@@ -85,16 +86,24 @@ async function RunTest(
   await connection.close();
 }
 
-async function ExecuteSendsAsync(sender: any, messages: number): Promise<void> {
+async function ExecuteSendsAsync(sender: Sender, messages: number): Promise<void> {
+  sender.setMaxListeners(1000);
```

Comment:
What does `setMaxListeners()` do, and is it relevant to this send-only scenario?

Reply:
I see this warning when `MaxListeners` is not set to a high number.
- Warning - `(node:31552) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 accepted listeners added. Use emitter.setMaxListeners() to increase limit`

- Example -  ![image](https://user-images.githubusercontent.com/10452642/56773960-70a1d780-6775-11e9-9048-ac2274c0acac.png)

- I believe there is a default limit on the number of listeners(SenderEvents.accepted, SenderEvents.rejected, etc) that can be added on a sender.

-------------------------------------------------

Verified this in the service-bus src code where we set `MaxListeners` to 1000.

https://github.com/Azure/azure-sdk-for-js/blob/b6797a49e2e2def2914258db0d271f611ca0feca/sdk/servicebus/service-bus/src/core/messageSender.ts#L657

To stay consistent with the service-bus, I've added `sender.setMaxListeners(1000);` in this rhea-sample and the warning vanished.